src/hooks.tsx

```diff
@@ -116,40 +116,49 @@ export const useAriaId = (id?: string): string => {
     }
 };
 
-export const useWindowSize = (): {
+const useSpecificWindowObject = (useInnerFrame: boolean): Window | undefined =>
+    typeof window !== 'undefined' ? (useInnerFrame ? window?.self : window) : undefined;
```

Comment:
I don't understand this logic, isn't `window === window.self` ?

Reply:
of course you don't understand, it was absolutely wrong and unnecessary! My fault, thanks for pointing out.